docs/blog/_posts/2017-07-11-second-dotty-milestone-release.md

```diff
@@ -0,0 +1,360 @@
+The highlights of this release are:
+ - substantial improvement of quality of generated code for pattern matching
+ - improvements in VS Code IDE stability
+ - improved compatibility with scalac 
+ - initial support for reproducible builds
+
+
+<!--more-->
+
+This is our second scheduled release according to our [6-week release schedule](http://dotty.epfl.ch/docs/usage/version-numbers.html).
+
+# What’s in the 0.2.0-RC1 technology preview?
+The [previous technology preview](/blog/2017/05/31/first-dotty-milestone-release.html) has shipped new language planned for Scala 3:  
+[Intersection Types](http://dotty.epfl.ch/docs/reference/intersection-types.html),
+[Union Types](http://dotty.epfl.ch/docs/reference/union-types.html),
+[Trait Parameters](http://dotty.epfl.ch/docs/reference/trait-parameters.html),
+[Enumerations](http://dotty.epfl.ch/docs/reference/enums/enums.html),
+[Algebraic Data Types](http://dotty.epfl.ch/docs/reference/enums/adts.html),
+[By-Name Implicits](http://dotty.epfl.ch/docs/reference/implicit-by-name-parameters.html).
+
+This technology preview is geared towards improving stability and reliability. It includes:
+
+  - [Local optimizations upstreamed from Dotty Linker](https://github.com/lampepfl/dotty/pull/2513), [2647](https://github.com/lampepfl/dotty/pull/2647) by ([@OlivierBlanvillain](https://github.com/OlivierBlanvillain). See more details below.
+  - [Optimizing Pattern Matcher](https://github.com/lampepfl/dotty/pull/2829) by ([@odersky](https://github.com/odersky)
+  - [Idempotency checks](https://github.com/lampepfl/dotty/pull/2756) are the first step to reproducible builds
+  - [Faster Base class sets](https://github.com/lampepfl/dotty/pull/2676) by ([@odersky](https://github.com/odersky) and ([@darkdimius](https://twitter.com/darkdimius) 
+  - Huge number of fixes to IDE and Dotty Language Server covering:
+   
+     - [Fix hover-on-type for implicitly converted expressions](https://github.com/lampepfl/dotty/pull/2836)
+     - [Fixes to find all references in external projects](https://github.com/lampepfl/dotty/pull/2810), [2773](https://github.com/lampepfl/dotty/pull/2773/files)
+     - [Fix conflict with  dragos-vscode-scala](https://github.com/lampepfl/dotty/pull/2777)
+     - [Fix ide crash on non-parsable file](https://github.com/lampepfl/dotty/pull/2752)
+     - [Fix hover functionality for enum classes](https://github.com/lampepfl/dotty/pull/2722)
+     - [Report errors on Dotty Language Server initialization](https://github.com/lampepfl/dotty/pull/2708)
+     - [Fixes to sbt setting up Dotty IDE](https://github.com/lampepfl/dotty/pull/2690)
+     - General stability improvements [2838](https://github.com/lampepfl/dotty/pull/2838), [2787](https://github.com/lampepfl/dotty/pull/2787), [2692](https://github.com/lampepfl/dotty/pull/2692)
+     
+  - Scalac compatibility improvements:
+  
+     - [Support Scala 2.12 traits](https://github.com/lampepfl/dotty/pull/2685)
```

Comment:
It's not actually working yet (https://github.com/lampepfl/dotty/pull/2827), so maybe not worth reporting yet.